modules/test/protocol/python/src/protocol_bacnet.py

```diff
@@ -54,23 +57,21 @@ def discover(self, local_ip=None):
 
   # Check if the device being tested is in the discovered devices list
   def validate_device(self, local_ip, device_ip):
-    result = None
     LOGGER.info('Validating BACnet device: ' + device_ip)
     self.discover(local_ip + '/24')
-    LOGGER.info('BACnet Devices Found: ' + str(len(self.devices)))
+    LOGGER.info('BACnet devices found: ' + str(len(self.devices)))
     if len(self.devices) > 0:
-      # Load a fail result initially and pass only
-      # if we can validate it's the right device responding
-      result = False, ('Could not confirm discovered BACnet device is the ' +
-                       'same as device being tested')
+      result = ('Feature Not Detected',
+                'Device did not respond to BACnet discovery')

```

Comment:
The reason I left this as Feature Not Detected is because I can see a use case (either now or in the future), where we allow users to connect the device testing interface to a switch with multiple devices. If there are other BACnet devices on that switch, that is fine - the user may have configured the device under test to communicate with peer devices using other protocols.

Reply:
I think until we have that option, we can't change the results of this test.  Once we have that ability we should re-evaluate how this is determined to account for multiple device connections as that scenario will be a sprawling change to almost all tests.

Reply:
Ok, I have reverted this one back to non-compliant.